plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -118,17 +118,17 @@ public JdbcMetadataConfig setDomainCompactionThreshold(int domainCompactionThres
     }
 
     @Min(1)
-    @Max(MAX_ALLOWED_INSERT_BATCH_SIZE)
-    public int getInsertBatchSize()
+    @Max(MAX_ALLOWED_WRITE_BATCH_SIZE)
+    public int getWriteBatchSize()
     {
-        return insertBatchSize;
+        return writeBatchSize;
     }
 
-    @Config("insert.batch-size")
+    @Config("write.batch-size")
```

Comment:
You can add `@LegacyConfig("insert.batch-size")` if this was already *released*.

Reply:
https://github.com/trinodb/trino/pull/8434 is in Milestone 360, but I'm not sure if it was already released.
@hashhar knows better for sure :)

Reply:
It's not in any released version yet - 360 will be the first release to include it.